libs/pages/services/src/lib/ui/page-settings-danger-zone/page-settings-danger-zone.tsx

```diff
@@ -0,0 +1,42 @@
+import { BlockContentDelete, HelpSection } from '@console/shared/ui'
+
+export interface PageSettingsDangerZoneProps {
+  deleteEnvironment: () => void
+}
+
+export function PageSettingsDangerZone(props: PageSettingsDangerZoneProps) {
+  const { deleteEnvironment } = props
+
+  return (
+    <div className="flex flex-col justify-between w-full">
+      <div className="p-8">
+        <BlockContentDelete
+          className="w-[580px]"
```

Comment:
Can we put this value in the theme? 
If one day we want to increase the size of all our boxes we could in one go with the tailwind theme